raiden/api/python.py

```diff
@@ -65,6 +65,32 @@ def get_channel(self, channel_address):
 
         raise ValueError("Channel not found")
 
+    def manager_address_if_token_registered(self, token_address):
+        """
+        If the token is registered then, return the channel manager address.
+        Returns None otherwise.
+        """
+        try:
+            manager = self.raiden.chain.manager_by_token(token_address)
+            return manager.address
+        except:
+            return None
+
+    def register_token(self, token_address):
+        """ Will register the token at `token_address` with raiden. If it's already
+        registered, will throw an exception."""
+        try:
+            channel_manager = self.raiden.chain.manager_by_token(token_address)
```

Comment:
Here it'll fail, if `manager_by_token` returns `''` instead of raising an exception.